app/main/controllers/template/rtmedia-ajax-actions.php

```diff
@@ -16,18 +17,50 @@ function rtmedia_delete_uploaded_media() {
 
 	if ( ! empty( $action ) && 'delete_uploaded_media' === $action && ! empty( $media_id ) ) {
 		if ( wp_verify_nonce( $nonce, 'rtmedia_' . get_current_user_id() ) ) {
+			$remaining_photos    = 0;
+			$remaining_music     = 0;
+			$remaining_videos    = 0;
+			$remaining_all_media = 0;
+			$rtmedia_media       = new RTMediaMedia();
+			$rtmedia_media->delete( $media_id );
+
+			// Fetch the remaining media count.
+			if ( class_exists( 'RTMediaNav' ) ) {
+				global $bp;
+				$rtmedia_nav_obj = new RTMediaNav();
+
+				if ( function_exists( 'bp_is_group' ) && bp_is_group() ) {
+					$counts = $rtmedia_nav_obj->actual_counts( $bp->groups->current_group->id, 'group' );
```

Comment:
If we're using any buddypress' variables, we must check `! empty` or `function_exists` conditions. What if buddypress changes this variable? It'll give fatal error.

Reply:
Condition check added for existing BuddyPress functions and variables.